index.bs

```diff
@@ -4017,11 +4019,15 @@ partial interface MLGraphBuilder {
     1. If |input|'s [=MLOperand/dataType=] is not {{MLOperandDataType/"float32"}} or {{MLOperandDataType/"float16"}}, then [=exception/throw=] a {{TypeError}}.
     1. If the [=MLOperand/rank=] of any of |input|, |weight| or |recurrentWeight| is not 3, then [=exception/throw=] a {{TypeError}}.
     1. If |input|'s [=MLOperand/shape=][0] is not equal to |steps|, then [=exception/throw=] a {{TypeError}}.
+<<<<<<< HEAD
```

Comment:
This seems to be a rebase issue.

Reply:
Thanks, fixed in 98e3892872e85cfec401624bb9ca1d178b3561f0